game/scripts/npc/abilities/tidehunter_anchor_smash.txt

```diff
@@ -47,7 +47,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage_reduction"                                "-30 -40 -50 -60 -60 -60"
+        "damage_reduction"                                "-30 -40 -50 -60 -70 -70"
```

Comment:
kinda weird that level 5 scales but level 6 doesn't :thinking:

Reply:
I did this because it would be nearly 100% damage reduction